upup/pkg/fi/cloudup/awstasks/launchtemplate_target_api.go

```diff
@@ -164,8 +166,19 @@ func (t *LaunchTemplate) RenderAWS(c *awsup.AWSAPITarget, a, e, changes *LaunchT
 			LaunchTemplateName: t.Name,
 			LaunchTemplateData: data,
 		}
-		if _, err = c.Cloud.EC2().CreateLaunchTemplateVersion(input); err != nil {
+		if version, err := c.Cloud.EC2().CreateLaunchTemplateVersion(input); err != nil {
 			return fmt.Errorf("error creating LaunchTemplateVersion: %v", err)
+		} else {
+			newDefault := strconv.FormatInt(*version.LaunchTemplateVersion.VersionNumber, 10)
+			_, err := c.Cloud.EC2().ModifyLaunchTemplate(
+				&ec2.ModifyLaunchTemplateInput{
+					DefaultVersion:   &newDefault,
+					LaunchTemplateId: version.LaunchTemplateVersion.LaunchTemplateId,
+				},
+			)
```

Comment:
Could we do this just for Karpenter? Users already complain about the amount of API calls.

Reply:
Users should not hit rate limits of this call at least. It's mostly non-mutating call that is a challenge, but these are separate buckets.

LTs do not know about karpenter, and I can't think of a reason why they would. I also can't think of how to inject such a value in a good way.

Reply:
OK, we can go with it as is and optimize later.